scripts/utilities/update_revision.sh

```diff
@@ -294,6 +316,9 @@ send_to_github ()
 	pkg=$(echo ${1} | awk -F "/" '{ print $(NF-1) }')
 	cfg=$(echo ${1} | awk -F "/" '{ print $(NF-3) }')
 
+        # Temp file to save output of REST calls to the GitHub API
+	out=$(mktemp '/tmp/ghapi-XXXXX.out')
```

Comment:
Isn't it important to remove this file at the end of the execution?

Reply:
I added a `trap` to guarantee it gets deleted.